clients/web/src/main.js

```diff
@@ -1,3 +1,30 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import Backbone from 'backbone';
+import moment from 'moment';
 import * as girder from 'girder';
 
 window.girder = girder;
+
+// Some cross-browser globals
+if (!window.console) {
+    window.console = {
+        log: $.noop,
+        error: $.noop
+    };
+}
+
+// For testing and convenience, available now because of testUtils.js reliance on $
+window.$ = $;
+window._ = _;
+window.moment = moment;
+window.Backbone = Backbone;
+
+if (!window.disableGirderMainApp) {
```

Comment:
I would prefer this stay in the mako template rather than the main bundle.  Is there a reason besides convenience for moving it here?